tests/network/unit/link_setup_test.py

```diff
@@ -37,9 +37,12 @@ def test_parse_nets_bridge_opts():
         },
     }
     expected = {
-        'br1': {'multicast_router': '0', 'multicast_snooping': '0'},
-        'br2': {'multicast_router': '1', 'multicast_snooping': '1'},
+        'br1': {'multicast-router': 0, 'multicast-snooping': False},
+        'br2': {'multicast-router': 1, 'multicast-snooping': True},
     }
 
-    for name, opts in linksetup.parse_nets_bridge_opts(nets):
-        assert expected[name] == opts
+    for name, opts in nets.items():
+        parsed_opts = linksetup.parse_nets_bridge_opts(
+            opts['custom']['bridge_opts']
```

Comment:
Is there a reason to keep the `custom` and `bridge_opts` nesting around?

Reply:
not sure what you mean here, can you elaborate?

Reply:
Since we are parsing only the string of the options now, we don't need to create the whole nets dictionary. The `nets` can be reduced to array same with expected.